sosia/classes/original.py

```diff
@@ -121,7 +121,7 @@ def __init__(
         self._matches = None
 
         # Create logger
-        log_path = Path(log_path) or DEFAULT_LOG
+        log_path = Path(log_path) if log_path else DEFAULT_LOG
```

Comment:
Otherwise if `log_path` is `None` we get an error

Reply:
You're right! Another option is:
```
log_path = log_path or DEFAULT_LOG
create_logger(Path(log_path))
```